runner/runners/combine.go

```diff
@@ -0,0 +1,44 @@
+package runners
```

Comment:
Suggest a name that conveys that is notable about this Runner.  Namely it's pluggable or controllable, so maybe "PluggableRunner", "ControllableRunner", or "FlexibleRunner", and then matching the file name.